frontend/src/store/modules/dashboard/actions.js

```diff
@@ -49,63 +52,28 @@ export default {
             });
     },
     [FETCHING_ACTIVITY_DATA_ITEMS]: (context) => {
-       const items = [
-           {
-               url:'link_1/juhy/kkk',
-               visitorId:2,
-               timeNotification:'2019-08-26 22:15:11'
-           },
-           {
-               url:'link_2/juhy/kkk',
-               visitorId:2,
-               timeNotification:'2019-08-26 23:25:11'
-           },
-           {
-               url:'link_2/juhy/kkk',
-               visitorId:3,
-               timeNotification:'2019-08-12 12:12:11'
-           },
-           {
-               url:'link_1/juhy/kkk',
-               visitorId:2,
-               timeNotification:'2019-08-12 12:19:11'
-           },
-           {
-               url:'link_2/juhy/kkk',
-               visitorId:2,
-               timeNotification:'2019-08-12 12:15:11'
-           },
-           {
-               url:'link_2/juhy/kkk',
-               visitorId:3,
-               timeNotification:'2019-08-12 12:11:11'
-           },
-           {
-               url:'link_1/juhy/kkk',
-               visitorId:3,
-               timeNotification:'2019-08-12 12:11:11'
-           },
-           {
-               url:'link_1/juhy/kkk',
-               visitorId:4,
-               timeNotification:'2019-08-12 12:11:11'
-           },
-           ].sort( (a, b) => {
-            return  a.timeNotification - b.timeNotification || a.url - b.url || a.visitorId - b.visitorId;
-       });
+        context.commit(SET_BUTTON_FETCHING);
 
-       const result = [];
-        items.forEach((element) => {
-            if(result.length > 0) {
-                if(!result.find( (item => item.url === element.url && item.visitorId === element.visitorId))) {
+        return getActivityDataItems().then(response => {
+            response.sort( (a, b) => {
+                return  a.date - b.date || a.url - b.url || a.visitor - b.visitor;
+            });
+
+            const result = [];
+            response.forEach((element) => {
+                if(result.length > 0) {
```

Comment:
in empty array you also won't find anything, so I think this condition doesn't have any sense